helios-client/src/main/java/com/spotify/helios/common/descriptors/DeploymentGroup.java

```diff
@@ -199,6 +222,9 @@ public Builder setName(final String name) {
       return this;
     }
 
+    // TODO(negz): Annotate this as @JsonProperty("job")?
+    // AFAICT we're currently serialising this with the key "jobId", despite annotating it as
+    // @JsonProperty("job") in the constructor and javadoc.
```

Comment:
Oh, right. Wrong `getJobId`. Yeah I was concerned about fixing it due to backwards compatibility. I could remove the TODO and just update the javadoc and leave a comment near the constructor annotation?